src/components/autocomplete/js/autocompleteController.js

```diff
@@ -527,6 +535,38 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Determines if the escape keydown should be processed
+   * @returns {boolean}
+   */
+  function shouldProcessEscape() {
+    if (shouldBlurOnEscape()) return true;                            // Needs blur
+    else if (!ctrl.hidden) return true;                               // Hide dropdown
+    else if (ctrl.loading) return true;                               // Cancel loading
+    else if (shouldClearOnEscape() && $scope.searchText) return true; // Clear Text
+    else return false;
+  }
```

Comment:
Oh, I know. I was just using `shouldHide` function as a model in case these it was part of the Style Guide